thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/bao/AlertManagerImpl.java

```diff
@@ -38,6 +38,16 @@ public Long loadValue() {
         return countActive();
       }
     });
+    metricRegistry.register("activeTimeseriesMonitoredCount", new CachedGauge<Integer>(15, TimeUnit.MINUTES) {
+      @Override
+      protected Integer loadValue() {
+        return findAllActive().stream()
+            // look for enumerationItems and get the item count
+            .map(alert -> ((List<?>)alert.getTemplateProperties().getOrDefault("enumerationItems", List.of())).size())
```

Comment:
This is fragile. one variable change and this metric breaks. 
now that we have alert inside enumeration item. just count the number of enumeration items per alert.